client/src/Movies/Movie.js

```diff
@@ -2,10 +2,10 @@ import React, { useState, useEffect } from 'react';
 import axios from 'axios';
 
 const Movie = (props) => {
-  const [movie, setMovie] = useState({});
+  const [movie, setMovie] = useState();
```

Comment:
You can also use `const [movie, setMovie] = useState([])` or you can use `const [movie, setMovie] = useState(null)`.